dashboard/app/helpers/levels_helper.rb

```diff
@@ -38,6 +38,17 @@ def create_channel(data = {}, src = nil)
     ))
     headers = result[1]
 
+    # The ChannelsApi may set a storage_id cookie that we need to copy to the response.
+    if headers['Set-Cookie']
+      cookie = headers['Set-Cookie'].split(';').map{ |token| token.strip.split('=') }
+      name = cookie[0][0]
+      cookie[0][0] = 'value'
+      cookie = Hash[cookie]
+      cookie['value'] = CGI.unescape cookie['value']
+      cookie['expires'] = Time.parse cookie['expires']
+      cookies[name] = cookie
```

Comment:
IIRC we originally went with the ChannelsApi.call approach because we were convinced by someone that "creating more APIs is bad" and that accessing StorageApps directly from Dashboard would be "creating more APIs". @joshlory please correct me if you remember differently :)